src/test/java/PlaybackTests.java

```diff
@@ -0,0 +1,22 @@
+import POM.Pages.HomePage;
+import POM.Pages.LoginPage;
+import POM.Pages.SongsPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class PlaybackTests extends BaseTest{
+
+    @Test
+    public void playSongDoubleClick() throws InterruptedException{
+        LoginPage loginPage = new LoginPage(getDriver());
+        SongsPage songsPage = new SongsPage(getDriver());
+
+        loginPage.login("angel.ayala@testpro.io", "school!sc0");
+        Thread.sleep(2000);
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.